tasks/build-website-testing.js

```diff
@@ -0,0 +1,8 @@
+var gulp = require('gulp');
+var config = require('./config.js');
```

Comment:
probaly is a bug in eslint with that rule (key-spacing align value)

to fix that, only opening a issue in eslint, I believe.

as this standard, I dont like this, because cause hard diffs in code-review/github, but it is just my opnion

for now, I will change all variables when eslint dont can validate, to follow the pattern